server/app.ts

```diff
@@ -1,12 +1,101 @@
 import express from 'express';
 
 export const app = express()
-const port = 3000
+export const port = 3000
+
+// For testing locally: https://www.npmjs.com/package/cors
+var cors = require('cors')
+app.use(cors())
+
+import { InfluxDB } from '@influxdata/influxdb-client'
+
+const token = 'kxF9oWgVQdrUMz3kFOwyVw8eRqr1guWFeuSVIdXlBi-8ANuuwH5utNX-bDqh1hbhbmQguXYnPNJCr7AwUONtVA=='
+const org = 'codebase-relativity'
+const bucket = 'relativity-ramp-up'
+const url = 'https://us-west-2-1.aws.cloud2.influxdata.com'
+
+const client = new InfluxDB({ url, token })
+const queryApi = client.getQueryApi(org)
+
 
 app.get('/', (req, res) => {
-  res.send('Hello World!')
+  res.send('Relativity Ramp Up Project')
+})
+
+app.get('/buckets', async (req, res) => {
+  const fluxQuery = 'buckets()'
+  res.json(await queryApi.collectRows(fluxQuery));
+})
+
+app.get('/altitude', async (req, res) => {
+  const fluxQuery = 'from(bucket: "relativity-ramp-up") |> range(start: 0) |> filter(fn: (r) => r._measurement == "altitude" )'
+  res.json(await queryApi.collectRows(fluxQuery));
+})
+
+app.get('/engines', async (req, res) => {
+  // https://docs.influxdata.com/influxdb/cloud/query-data/flux/regular-expressions/
+  
+  // Memory allocation limit reached:
+  // const fluxQuery = 'from(bucket: "relativity-ramp-up") |> range(start: 0) |> filter(fn: (r) => r._measurement =~ /engine-[1-3]$/ ) |> aggregateWindow(every: 1s, fn: mean)'
```

Comment:
Oh no ;-; i think someone else did this and it worked? We shouldn't run into this on the real thing though!